studio/components/grid/components/header/RLSBannerWarning.tsx

```diff
@@ -29,7 +29,7 @@ export default function RLSBannerWarning() {
     <>
       {!isAcknowledged && !rlsEnabled && isPublicTable ? (
         <div>
-          <div className="text-center bg-amber-500 text-amber-900 text-xs py-2.5 flex items-center justify-center relative">
+          <div className="text-center bg-amber-500 text-scale-1200 text-xs py-2.5 flex items-center justify-center relative">
```

Comment:
you can try `text-amber-1000` \ `text-amber-1100`

they will still be amber, but easier to read

Reply:
might need `"text-amber-1000 dark:text-amber-900"`